core/src/main/java/hudson/model/UsageStatisticsContributorForCore.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * The MIT License
+ * 
+ * Copyright (c) 2004-2016, Sun Microsystems, Inc., Kohsuke Kawaguchi, Baptiste Mathus
+ * 
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * 
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import hudson.Extension;
+import hudson.PluginWrapper;
+import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
+
+@Extension
+public class UsageStatisticsContributorForCore extends UsageStatisticsContributor {
```

Comment:
Yeah. Renamed. 
Originally named it to have it visually alongside UsageStatistics while starting the modifications. I admit it was not great.